cmd/contour/serve.go

```diff
@@ -155,6 +155,9 @@ func registerServe(app *kingpin.Application) (*kingpin.CmdClause, *serveContext)
 
 	serve.Flag("debug", "Enable debug logging.").Short('d').BoolVar(&ctx.Config.Debug)
 	serve.Flag("kubernetes-debug", "Enable Kubernetes client debug logging with log level.").PlaceHolder("<log level>").UintVar(&ctx.KubernetesDebug)
+
+	serve.Flag("gateway-class-name", "Contour GatewayClass name.").PlaceHolder("<api_group>/<namespace>/contour").StringVar(&ctx.gatewayClassName)
```

Comment:
If we do find we need this then it should come from the config file, not a flag. We've been adding all those bits to the config file that needs configuring.

Reply:
> From the last meeting, was this intended to be convention?

@stevesloka from my understanding `<api_group>/<namespace>/contour` was the agreed upon convention.

> The GatewayClass name would match how Contour is deployed? 

Yes, the namespace that Contour is deployed.

> Also should replace <api_group> with projectcontour.io since that's what's expected.

I used <api_group> since this portion of the string is derived from the API group. I'll update to the group string.

> If we do find we need this then it should come from the config file, not a flag.

Per feedback from the meeting, I was following how the ingress class name is configured. I'll update to use the config file.